examples/publication/frontend/best-practice-regulation.json

```diff
@@ -1,7 +1,7 @@
 {
   "analytics_identifier": null,
   "base_path": "/government/publications/regulatory-article-ra-2135-medical-requirements",
-  "content_id": "5fecc60b-7631-11e4-a3cb-005056011aef",
+  "content_id": "624666b3-cbdc-4622-8820-5dde7e0059e5",
```

Comment:
Why has the content_id changed?

Reply:
It was the same as `best-practice-guidance` which I thought was causing problems but it wasn't so I'll revert. 👍